packages/taquito-core/src/error/common.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ *  @category Error
+ *  @description Error that indicates an invalid address being passed or used (both contract and implicit)
+ */
+export class InvalidAddressError extends Error {
```

Comment:
Unsure I understand the context correctly, I thought we plan to extend common errors by higher level errors created by task1?

Reply:
this is just moving the current error to common. we can change what it extends after this ticket

Reply:
I see, so taquito/core will also be one of the package that we'll work on ticket 1994. Since common.ts will be used across packages do we want to prioritize as first 5 that we'll tackle?

Reply:
yes because any modifications of locations or additions of higher level functions will add to taquito/core